content/10.diagnostics.md

```diff
@@ -51,15 +72,17 @@ For example, PCR-based test can be highly sensitive, but in high-throughput sett
 3. Type of specimen, as it is not clear which clinical samples are best to detect the virus [@doi:10.1038/d41587-020-00002-2].
 4. Expensive machinery, which might be present in major hospitals and/or diagnostic centers but is often not available to smaller facilities [@doi:10.1126/science.abb8400].
 5. Timing of the test, which might take up to 4 days to give results [@doi:10.1126/science.abb8400].
+6. The availability of supplies for testing, including swabs and testing media, has been limited [@doi:10.1128/JCM.00512-20].
 
 False positives are a concern with this technique, as the guide RNA can recognize other interspersed sequences on the patient’s genome, resulting in a loss of specificity.
```

Comment:
```suggestion
```
Per @matfax suggestion

Reply:
This is weird -- I tried to move this up to add it to the list per @matfax suggestion, not strike it completely! Not sure what happened. Definitely leave it in!